site/en/docs/webstore/best_practices/index.md

```diff
@@ -54,162 +76,31 @@ See [Supplying Images][8] for guidelines on the images you should supply to the
-      </td>
-    </tr>
-    <tr>
-      <td>Games</td>
-      <td>
-        Action & Adventure Games<br>
-        Arcade Games<br>
-        Board Games<br>
-        Card Games<br>
-        Educational Games<br>
-        Puzzle Games<br>
-        Role Playing Games<br>
-        Sports Games<br>
-        Strategy Games<br>
-        Virtual Worlds
-      </td>
-      <td>All kinds of games that users can play. Also see Education and Entertainment.</td>
-    </tr>
-    <tr>
-      <td>Lifestyle</td>
-      <td>
-        Astrology<br>
-        Fashion<br>
-        Food<br>
-        Health & Fitness<br>
-        Money<br>
-        Religion<br> 
-        Shopping<br>
-        Travel
-      </td>
-      <td>Apps for everyday life.</td>
-    </tr>
-    <tr>
-      <td>News & Weather</td>
-      <td>
-        Blogs<br>
-        Magazines<br>
-        News Aggregators<br>
-        Newspapers<br>
-        Social News<br>
-        Sports<br>
-        Weather Forecasts
-      </td>
-      <td>Apps that feature news about current events or weather.</td>
-    </tr>
-    <tr>
-      <td>Productivity</td>
-      <td>
-        Calendar & Scheduling<br>
-        Creative Tools<br>
-        Design Essentials<br> 
-        Developer Tools<br>
-        Office Applications<br>  
-        Online Documents & File Storage<br>
-        Search & Browsing Tools<br>
-        Task Management
-      </td>
-      <td>
-        Apps that someone might spend a significant amount of time using to get something done. 
-        Also see Utilities.
-      </td>
-    </tr>
-    <tr>
-      <td>Social & Communication</td>
-      <td>
-        Blogging<br>
-        Chat & IM<br>
-        Dating<br>
-        Email & Contacts<br>
-        Phone & SMS<br>
-        Social Networking
-      </td>
-      <td>Apps that help people communicate with each other.</td>
-    </tr>
-    <tr>
-      <td>Utilities</td>
-      <td>
-        Alarms & Clocks<br>
-        Bookmarks<br>
-        Calculators<br>
-        Dictionaries<br>
-        Notepads
-      </td>
-      <td>Useful apps that have a narrower scope than the ones in the Productivity category.</td>
-    </tr>
-  </tbody>
-</table>
-
-[1]: /docs/apps
-[2]: /docs/webstore/launching#pre-launch-checklist
+The developer console lets you specify a category for each extension. Choose the category
+that is most appropriate:
+
+* Accessibility
+* Blogging
+* Developer Tools
+* Fun
+* News & Weather
+* Photos
+* Productivity
+* Search Tools
+* Shopping
+* Social & Communication
+* Sports
+
+
 [3]: /docs/webstore/identify_user
 [4]: /docs/webstore/check_for_payment
 [5]: /docs/webstore/check_for_payment#cache
 [6]: #top
 [7]: /docs/webstore/branding
 [8]: /docs/webstore/images
 [9]: #top
+[mv3-overview]: /docs/extensions/mv3/intro/mv3-overview/
+[program policies]: /docs/webstore/program_policies/
+[user-daat-faq]: /docs/webstore/user_data/
+[supplying-images]: /docs/webstore/images/
+[completing-listing]: /docs/webstore/cws-dashboard-listing/
```

Comment:
```suggestion
[completing-listing]: /docs/webstore/cws-dashboard-listing/
[dashboard-privacy]: /docs/webstore/cws-dashboard-privacy/
[mv3-overview]: /docs/extensions/mv3/intro/mv3-overview/
[program policies]: /docs/webstore/program_policies/
[supplying-images]: /docs/webstore/images/
[user-data-faq]: /docs/webstore/user_data/
```

Fixes a typo in "user-data-faq", adds a "dashboard-privacy" link,  and sorts the links alphabetically.